dao/src/main/java/com/epam/dao/impl/DepartmentDaoImpl.java

```diff
@@ -58,13 +60,13 @@ public DepartmentDaoImpl(NamedParameterJdbcTemplate namedParameterJdbcTemplate)
         return Optional.ofNullable(DataAccessUtils.uniqueResult(namedParameterJdbcTemplate.query(selectById, map, new DepartmentMapper())));
     }
 
-    public Department update(Department department, Integer id) throws Exception {
+    public int update(Department department, Integer id) {
```

Comment:
Please remove id from input parameters, it can be provided as department property